Launching-new-features.md

```diff
@@ -311,21 +311,21 @@ if (this.featureService.status.NewFeature.isEnabled) {
 
 ## Changing Value of Feature Flags
 
-Feature flags are defaulted to `false/disabled`. To change their values, you can login as the administrator, navigate to the admin page, then to the feature tab.
+Feature flags are defaulted to `false/disabled`. To change their values, you can login as the administrator, provide yourself the role of release-coordinator from the 'Roles' tab present on the Admin page, navigate to the release-coordinator page, then to the feature tab.
 
 In the feature tab, where you will see the feature flag you added, you can change the settings (see the [Setting of Feature Flags](#settings-of-feature-flags) section for detail) of the feature flags.
 
-Note: since only users with admin permission can edit the settings of feature flags, you can only enable your features on your local dev instance of Oppia, while in production environment, only administrators can enable/disable features.
+Note: since only users with release-coordinator permission can edit the settings of feature flags, you can only enable your features on your local dev instance of Oppia, while in production environment, only release-coordinators can enable/disable features.
```

Comment:
The second of these should have no hyphen.

Let's also break this up a bit: ...of Oppia. However, on the live site, only release coordinators ...

Reply:
Done